teuthology/lock/ops.py

```diff
@@ -229,9 +233,15 @@ def update_lock(name, description=None, status=None, ssh_pub_key=None):
 
     if updated:
         uri = os.path.join(config.lock_server, 'nodes', name, '')
-        response = requests.put(
-            uri,
-            json.dumps(updated))
+        inc = random.uniform(0, 1)
```

Comment:
Why this is needed?

Reply:
The idea is to spread out retries in case of multiple conflicts resulting in the same retry interval, and encountering a conflict again.